build.gradle

```diff
@@ -72,6 +72,12 @@ configure(subprojects.findAll { it.name != 'integration-test' }) {
         from sourceSets.main.allSource
     }
 
+    task fatJar(type: Jar) {
```

Comment:
What is this for?

Reply:
This task make jar contains with all dependency lib. It needs to develop caver-java-ext-kas. 
If you think this task don't need, I will delete this task.